index.d.ts

```diff
@@ -0,0 +1,438 @@
+import EmberObject from '@ember/object';
+import {
+  UnwrapComputedPropertyGetter,
+  UnwrapComputedPropertyGetters
+} from '@ember/object/-private/types';
```

Comment:
These private types change _all the time_, and are often in need of adjustment to respond to breaking TS compiler changes. _**Please do not consume these directly**_